test/e2e/network/ingress.go

```diff
@@ -651,6 +586,14 @@ var _ = SIGDescribe("Loadbalancing: L7", func() {
 				jig.WaitForIngress(false /*waitForNodePort*/)
 			}
 		})
+
+		It("should create ingress with pre-shared certificate", func() {
+			executePresharedCertTest(f, jig, ipName)
+		})
+
+		It("should create ingress with backside re-encryption", func() {
```

Comment:
probably add some comment on what do you mean by backside re-encryption?
In our docs, we refer to it as "Backend HTTPS": https://github.com/kubernetes/ingress-gce/blob/master/README.md#backend-https